iotdb-client/service-rpc/pom.xml

```diff
@@ -109,7 +109,7 @@
         <dependency>
             <groupId>org.apache.iotdb</groupId>
             <artifactId>tsfile</artifactId>
-            <version>${project.version}</version>
+            <version>1.3.0-SNAPSHOT</version>
```

Comment:
Why do you change `${project.version}` to `1.3.0-SNAPSHOT`?

Reply:
Well actually that's a best practice ... the release plugin replaces all of them so that's no extra work.

Reply:
If I want using `mvn versions:set -DnewVersion=xxx` to change the project version, will the version here also change to the new one?